apollo-router/src/lib.rs

```diff
@@ -55,6 +55,7 @@ mod apollo_studio_interop;
 pub(crate) mod axum_factory;
 mod batching;
 mod cache;
+#[macro_use]
```

Comment:
Delete this line.

Reply:
You’re right, this was for a previous iteration where I had a custom `Result`-like type and a `try!-like macro (with the role of the `?` operator). This did not pan out because Tower really wants `Result`.